vision/google/cloud/vision/geometry.py

```diff
@@ -73,20 +87,35 @@ def __init__(self, latitude, longitude):
         self._longitude = longitude
 
     @classmethod
-    def from_api_repr(cls, response):
+    def from_api_repr(cls, location_info):
         """Factory: construct location information from Vision API response.
 
-        :type response: dict
-        :param response: Dictionary response of locations.
+        :type location_info: dict
+        :param location_info: Dictionary response of locations.
 
         :rtype: :class:`~google.cloud.vision.geometry.LocationInformation`
         :returns: ``LocationInformation`` with populated latitude and
                   longitude.
         """
-        latitude = response['latLng']['latitude']
-        longitude = response['latLng']['longitude']
+        lat_long = location_info.get('latLng')
+        latitude = lat_long.get('latitude')
+        longitude = lat_long.get('longitude')
```

Comment:
The  constructor looks like this, so it will make the class where either `LocationInformation.latitude` or `LocationInformation.longitude` is `None.

```python
def __init__(self, latitude, longitude):
    self._latitude = latitude
    self._longitude = longitude
```

Should I add `None` as a possible value to the docstrings?